interfaces/udev/udev.go

```diff
@@ -45,7 +72,7 @@ func (b *Backend) reloadRules(subsystemTriggers []string) error {
 	// By default, trigger for all events except the input subsystem since
 	// it can cause noticeable blocked input on, for example, classic
 	// desktop.
-	output, err = exec.Command("udevadm", "trigger", "--subsystem-nomatch=input").CombinedOutput()
+	output, err = exec.Command("udevadm", maybeTriggerQuiet("--subsystem-nomatch=input")...).CombinedOutput()
```

Comment:
(bikeshed) My concern would be a bit that the next time we add code in a different place of the file we may have forgoten that instead of just passing "--trigger" we use maybeTrigger(). So maybe make it all a bit more structured by e instead have something a bit like we do with systemd? Strawman:
```
struct udevadm {}
func (u *udevadm) Trigger(args)
func (u *udevadm) Control(args)
```

but maybe overkill because:
a) udev.go is very small so it's hard to not see this
b) quite a bit more unneeded churn

Could be a followup of course.

Reply:
Fair point, might be worth considering (maybe in a followup indeed). No strong opinion but I'm open to consider this idea 👍

Reply:
Yeah, like I wrote, maybe overkill given the counter points I had there.